From 6df2642ed7d300013229a28ed81c61457eacdd25 Mon Sep 17 00:00:00 2001
From: Peng Zhou <benjo.zhou@rock-chips.com>
Date: Thu, 16 Nov 2017 17:20:03 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3368-android: move cif-sensor from
 dtsi to dts

Move cif-sonsor from rk3368-android.dtsi to rk3368-px5-evb-android.dts

Change-Id: If3c0f8db0e15b2d28ef18d7c48e6e63eb2c00d47
Signed-off-by: Peng Zhou <benjo.zhou@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3368-android.dtsi        | 2 --
 arch/arm64/boot/dts/rockchip/rk3368-px5-evb-android.dts | 1 +
 2 files changed, 1 insertion(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3368-android.dtsi b/arch/arm64/boot/dts/rockchip/rk3368-android.dtsi
index 9f2acf90d268..83a08ce7cea6 100644
--- a/arch/arm64/boot/dts/rockchip/rk3368-android.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3368-android.dtsi
@@ -40,8 +40,6 @@
  *     OTHER DEALINGS IN THE SOFTWARE.
  */
 
-#include "rk3368-cif-sensor.dtsi"
-
 / {
 	chosen {
 		bootargs = "earlycon=uart8250,mmio32,0xff690000 swiotlb=1 firmware_class.path=/system/vendor/firmware";
diff --git a/arch/arm64/boot/dts/rockchip/rk3368-px5-evb-android.dts b/arch/arm64/boot/dts/rockchip/rk3368-px5-evb-android.dts
index d6a851fd3b89..503604db82ee 100644
--- a/arch/arm64/boot/dts/rockchip/rk3368-px5-evb-android.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3368-px5-evb-android.dts
@@ -43,6 +43,7 @@
 /dts-v1/;
 #include "rk3368.dtsi"
 #include "rk3368-android.dtsi"
+#include "rk3368-cif-sensor.dtsi"
 #include <dt-bindings/pwm/pwm.h>
 #include <dt-bindings/sensor-dev.h>
 
-- 
2.35.3

